check_sites.py

```diff
@@ -111,10 +115,17 @@ def main():
                 return
         old_checker = RwsCheck(old_sites, etlds, icanns)
         check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), rws_checker.load_sets())
-        # TODO: add variable and check for subtracted_sets in case of user 
-        # removing old set from the list
     else:
         check_sets = rws_checker.load_sets()
+        if testing_sets:
+            temp_sets = {}
+            for testing_set in testing_sets:
+                if testing_set not in check_sets:
+                    error_texts.append("There was an error loading the set:\n" + testing_set +
+                        " could not be found in related_website_sets.JSON")
+                else:
+                    temp_sets.update({testing_set: check_sets[testing_set]})
```

Comment:
FYI, using `.update` is overkill here:

```suggestion
                    temp_sets[testing_set] = check_sets[testing_set]
```